app/charges/controllers.py

```diff
@@ -239,6 +245,12 @@ def create_charge(user, user_data):
 def edit_charge(user, user_data):
     charge = Charges.query.filter_by(id = user_data.get("charge",-1)).first()
 
+    # The charge title must be at least 2 characters long and cannot contain special characters
+    invalid_chars = r'/^()#@![]{}`~\?%*:|"<>.'
+    if not user_data['title'] or len(user_data['title']) <= 1 or any(char in user_data['title'] for char in invalid_chars):
```

Comment:
This may be a nitpick but does `not user_date['title']` here have the same behavior as `user_data['title'] is None` above?

Reply:
I meant to use  not user_data['title'] in both places thank you. When the user enters a blank title 'not' catches this while 'is None' does not for some reason. Something to do with the JS to Python translation i think